pkg/sfu/utils/wraparound.go

```diff
@@ -211,6 +211,6 @@ func (w *WrapAround[T, ET]) isWrapBack(earlier T, later T) bool {
 
 // ------------------------------------
 
-func getExtendedHighest[T number, ET extendedNumber](cycles ET, val T) ET {
+func getExtended[T number, ET extendedNumber](cycles ET, val T) ET {
```

Comment:
drive-by renaming. The `Highest` does not make sense here. It is just calculating extended.